### DISCUSSION TITLE
kill two birds with one stone ?

### DISCUSSION THREAD

**Discussion by tiagonmas** [banco-alimentar/alimentestaideia.pt#246] [Ideas]
We want to remake the donation part of www.bancoalimentar.pt that currently does not produce invoices automatically, and payments are done via ifthenpay. My thought is to create a new website doar.bancoalimentar.pt that implements the payment and invoice features like we have in alimentestaideia.pt

Since the sites are so similar, one thought that came to mind was to use the same code base, but have 2 UI's for each site.
This way we would share the most important part, and when we implement new features (like recurring payments) they would be available in both sites ?!

thoughts @guerrerotook @joalmeid  ?

**Comment by guerrerotook** [comment]
Well, we can make a version of the current page with just one ProductCatalogue, cash and remove the rest of the pages.

**Reply by tiagonmas** [reply]
but that would require us to fork and maintain two different solutions. My thought was to have just one source code for both :)
